### PR TITLE
Add table aws_lambda_function_metric_duration_daily. Closes #623

### LINKED ISSUE #623: Add table aws_lambda_function_metric_duration_daily
**References**
Add any related links that will help us understand the resource, including vendor documentation, related GitHub issues, and Go SDK documentation.

Can be used in AWS_thrifty mod.

### PR DESCRIPTION
**Pull request by c0d3r-arnab**
# Example query results
<details>
  <summary>Results</summary>
  
```
> select
  name,
  timestamp,
  minimum,
  maximum,
  average,
  sample_count
from
  aws_lambda_function_metric_duration_daily
order by
  name,
  timestamp;
+--------------------------+---------------------+---------+---------+--------------------+--------------+
| name                     | timestamp           | minimum | maximum | average            | sample_count |
+--------------------------+---------------------+---------+---------+--------------------+--------------+
| arnab-hello-world-python | 2021-08-22 12:00:00 | 1.31    | 2.16    | 1.5566666666666666 | 6            |
| func-encrypted           | 2021-06-30 12:00:00 | 0.86    | 100.62  | 2.0349537037037035 | 216          |
| function-without-vpc     | 2021-08-22 12:00:00 | 13.2    | 13.2    | 13.2               | 1            |
+--------------------------+---------------------+---------+---------+--------------------+--------------+
> select
  name,
  timestamp,
  round(minimum::numeric,2) as min_duration,
  round(maximum::numeric,2) as max_duration,
  round(average::numeric,2) as avg_duration,
  sample_count
from
  aws_lambda_function_metric_duration_daily
where maximum > 100
order by
  name,
  timestamp;
+----------------+---------------------+--------------+--------------+--------------+--------------+
| name           | timestamp           | min_duration | max_duration | avg_duration | sample_count |
+----------------+---------------------+--------------+--------------+--------------+--------------+
| func-encrypted | 2021-06-30 12:00:00 | 0.86         | 100.62       | 2.03         | 216          |
+----------------+---------------------+--------------+--------------+--------------+--------------+
> select
  name,
  timestamp,
  round(minimum::numeric,2) as min_duration,
  round(maximum::numeric,2) as max_duration,
  round(average::numeric,2) as avg_duration,
  sample_count
from
  aws_lambda_function_metric_duration_daily
where average < 5
order by
  name,
  timestamp;
+--------------------------+---------------------+--------------+--------------+--------------+--------------+
| name                     | timestamp           | min_duration | max_duration | avg_duration | sample_count |
+--------------------------+---------------------+--------------+--------------+--------------+--------------+
| arnab-hello-world-python | 2021-08-22 12:00:00 | 1.31         | 2.16         | 1.56         | 6            |
| func-encrypted           | 2021-06-30 12:00:00 | 0.86         | 100.62       | 2.03         | 216          |
+--
```
</details>

### CODE REVIEW

File: docs/tables/aws_lambda_function_metric_duration_daily.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Table: aws_lambda_function_metric_duration_daily
+
+Amazon CloudWatch Metrics provide data about the performance of your systems.  The `aws_lambda_function_metric_duration_daily` table provides metric statistics at 24 hour intervals for the last year.
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_duration_daily.md:3]:
```suggestion
Amazon CloudWatch Metrics provide data about the performance of your systems. The `aws_lambda_function_metric_duration_daily` table provides metric statistics at 24 hour intervals for the last year.
```

File: docs/tables/aws_lambda_function_metric_duration_daily.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_duration_daily.md:4]:
```suggestion
```

**Review by bigdatasourav — CHANGES_REQUESTED** [review]
@c0d3r-arnab few small change, otherwise looks good.

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**